.github/workflows/build_push_dev_img.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           aws-region: us-west-2
       - name: Start EC2 runner
         id: start-ec2-runner
-        uses: machulav/ec2-github-runner@v2
+        uses: machulav/ec2-github-runner@v3
```

Comment:
@farshidz , you mentioned that we should ping this to a commit hash. Should we ping it to the commit of v2 version we used previously or the commit of the latest release version?